pkgs/fastfetch/default.nix

```diff
@@ -0,0 +1,113 @@
+# Credits going to federicoschonborn, taken from NUR and adapted to -git
+{ cmake
+, fetchFromGitHub
+, lib
+, nyxUtils
+, pkg-config
+, stdenv
+, enableLibpci ? false
+, pciutils
+, enableVulkan ? false
+, vulkan-loader
+, enableWayland ? false
+, libffi
+, wayland
+, enableXcb ? false
+, enableXrandr ? false
+, enableX11 ? false
+, xorg
+, enableGio ? false
+, glib
+, libselinux
+, libsepol
+, pcre16
+, pcre2
+, utillinux
+, enableDconf ? false
+, dconf
+, enableDbus ? false
+, dbus
+, enableXfconf ? false
+, xfce
+, enableSqlite3 ? false
+, sqlite
+, enableRpm ? false
+, rpm
+, enableImagemagick7 ? false
+, imagemagick
+, enableZlib ? false
+, zlib
+, enableChafa ? false
+, chafa
+, enableEgl ? false
+, libGL
+, enableGlx ? false
+, libglvnd
+, enableMesa ? false
+, mesa_drivers
+, enableOpencl ? false
+, ocl-icd
+, opencl-headers
+, enableLibcjson ? false
+, cjson
+, enableLibnm ? false
+, networkmanager
+, enableFreetype ? false
+, freetype
+, enablePulse ? false
+, pulseaudio
+,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "fastfetch";
+  version = "unstable-2023-05-03";
+
+  src = fetchFromGitHub {
+    owner = "LinusDierheimer";
+    repo = "fastfetch";
+    rev = "4d7cb25aa61e5cd06779e126046267edb062cd49";
+    hash = "sha256-YEW0AoplLx83gCd2ualyagsKm6CF1JND+FSllvbq7vs=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+  ];
+
+  buildInputs =
+    [ ]
+    ++ lib.optionals enableChafa [ chafa ]
+    ++ lib.optionals enableDbus [ dbus ]
+    ++ lib.optionals enableDconf [ dconf ]
+    ++ lib.optionals enableEgl [ libGL ]
+    ++ lib.optionals enableFreetype [ freetype ]
+    ++ lib.optionals enableGio [ glib libselinux libsepol pcre16 pcre2 utillinux ]
+    ++ lib.optionals enableGlx [ libglvnd ]
+    ++ lib.optionals enableImagemagick7 [ imagemagick ]
+    ++ lib.optionals enableLibcjson [ cjson ]
+    ++ lib.optionals enableLibnm [ networkmanager ]
+    ++ lib.optionals enableLibpci [ pciutils ]
+    ++ lib.optionals enableMesa [ mesa_drivers.dev ]
+    ++ lib.optionals enableOpencl [ ocl-icd opencl-headers ]
+    ++ lib.optionals enablePulse [ pulseaudio ]
```

Comment:
Is `libpulseaudio` enough?

Reply:
`fastfetch-unstable> [ 91%] Building C object CMakeFiles/libfastfetch.dir/src/detection/sound/sound_linux.c.o`
Looks like it is?